src/popup/pages/MultisigProposalDetails.vue

```diff
@@ -0,0 +1,672 @@
+<template>
```

Comment:
Is there a possibility to extract shared part of this view and standard TransactionDetails to a separate component?